app-frontend/src/assets/styles/sass/components/_searchselect.scss

```diff
@@ -72,9 +73,19 @@
   z-index: 1;
   box-sizing: border-box;
   border-radius: 3px;
-  border-color: $shade-dark;
-  background-color: $shade-dark;
-  color: $gray-lightest;
+  border-color: $shade-light;
```

Comment:
The style changes look good on the scene detail modal, but this also causes the browse datasource filter to have a light background. This may be fine but we may want to ask @designmatty. We could always have a dark and light class for the dropdown.

Reply:
pretty sure it was supposed to be a light background in the first place, to match the design of the drawing dropdown
![image](https://user-images.githubusercontent.com/4392704/36109339-18853474-0fed-11e8-9c8c-4bd0c53ec11a.png)

Reply:
🤷‍♂️